.travis.yml

```diff
@@ -12,6 +12,17 @@ after_success:
     - docker run --name patterns-core liberoadmin/patterns-core node_modules/.bin/gulp exportPatterns
     # TODO: empty out committed export/ folder
     - docker cp patterns-core:/patterns-core/export/. export/
+    - .travis/package.sh
```

Comment:
Could put this and the `deploy` in a different stage? Will be a little slower.

Reply:
Or use `before_deploy`?

Reply:
I wanted to always execute the packaging, so that you get any failure in a PR rather than only when merging. A different stage could work for this.